salt/states/btrfs.py

```diff
@@ -0,0 +1,258 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Alberto Planas <aplanas@suse.com>
+#
+# Copyright 2018 SUSE LINUX GmbH, Nuernberg, Germany.
```

Comment:
Copyright should be removed. Also, the windows test failures need to be addressed.